### ISSUE TITLE
shipping price import?

### ISSUE THREAD

**Issue by jojouser** [microweber/microweber#295]
Can we add shipping cost from csv(or other format) file?
Now we can add every country's shipping price in adminpanel, but if we can import shipping price from table file,it will be much quicker. thanks.

**Comment by ash-rain** [comment]
I agree it would be useful but isn't that too project-specific of a feature to include in the Microweber core?
You're free to extend Microweber in any way you see fit. Also, you can create a pull request later so that we can include custom functionality if it makes sense to the majority of people using it.

**Comment by Dalboz** [comment]
The option to maintain this kind of information for few items is ok, but the idea is interesting, but in a broad perspective, because for example I imagine a  store trying to couple their Stock/Finance/ERP systems within their web portal (microweber), and it looks scary to maintain.

The solution to update information fields like prices, currencies, quantity available,sku's, characteristics, etc seems more flexible and very useful for this scenario, that is, where are involved large amount of items eased by just uploading a plain text file.

**Comment by Dalboz** [comment]
The option to maintain this kind of information for few items is ok, but the idea is interesting, but in a broad perspective, because for example I imagine a  store trying to couple their Stock/Finance/ERP systems within their web portal (microweber), and it looks scary to maintain.

The solution to update information fields like prices, currencies, quantity available,sku's, characteristics, etc seems more flexible and very useful for this scenario, that is, where are involved large amount of items eased by just uploading a plain text file.

**Comment by ash-rain** [comment]
It makes sense. Are you interested in getting in touch on PM about this? You seem like you have good ideas.

**Comment by Dalboz** [comment]
I think a good example of this kind of functionality (import / export data) is represented in https://github.com/PrestaShop/PrestaShop
